scylla-types: add serialize action (#12029)
scylladb/scylladb: docs/operating-scylla/admin-tools/scylla-types.rst

.. code-block:: console

   scylla types deserialize --prefix-compound -t TimeUUIDType -t Int32Type 0010d00819896f6b11ea00000000001c571b000400000010

.. _scylla-types-operations:

Supported Operations
^^^^^^^^^^^^^^^^^^^^^^^

> * ``serialize`` - Serialize the value and print it in a hex encoded form. Required arguments: 1 value in human-readable form. To avoid problems around special symbols, separate values with ``--`` from the rest of the arguments.

Reviewer: ```suggestion
* ``serialize`` - Serializes the value and prints it in a hex-encoded form. Required arguments: 1 value in a human-readable form. To avoid problems around special symbols, separate the values from the rest of the arguments with ``--``.
```

* ``deserialize`` - Deserializes and prints the provided value in a human-readable form. Required arguments: 1 or more serialized values.

> * ``compare`` - Compares two values and print the result. Required arguments: 2 serialized values.

Reviewer: ```suggestion
* ``compare`` - Compares two values and prints the result. Required arguments: 2 serialized values.
```

* ``validate`` - Verifies if the value is valid for the type, according to the requirements of the type. Required arguments: 1 or more serialized values.
* ``tokenof`` - Calculates the token of the partition key (i.e. decorates it). Required arguments: 1 or more serialized values. Only accepts partition keys (``--full-compound``).

> * ``shardof`` - Calculates the token of the partition key and the shard it belongs to, given the provided shard configuration (``--shards`` and ``--ignore-msb-bits``). In most cases only ``--shards`` has to be provided, unless you have a non-standard configuration. Required arguments: 1 or more serialized values. Only accepts partition keys (``--full-compound``).

Reviewer: ```suggestion
* ``shardof`` - Calculates the token of the partition key and the shard it belongs to, given the provided shard configuration (``--shards`` and ``--ignore-msb-bits``). In most cases, only ``--shards`` has to be provided unless you have a non-standard configuration. Required arguments: 1 or more serialized values. Only accepts partition keys (``--full-compound``).
```

> You can learn more about each operation by invoking the help of said operation:

Reviewer: You need to add an empty line here for the paragraph to render correctly.
